src/plugins/data_table/data_table.js

```diff
@@ -2,30 +2,167 @@ var LocalDataTable = (function() {
 
   var Base = Backdraft.plugin("Base");
 
+  var DEFAULT_JST_DELIMS = {
+    evaluate    : /<%([\s\S]+?)%>/g,
+    interpolate : /<%=([\s\S]+?)%>/g,
+    escape      : /<%-([\s\S]+?)%>/g
+  };
+
+  var DataTableFilter = Base.View.extend({
+    filterTemplate : _.template('\
+      <div class="toggle-filter-button" data-toggle="dropdown">\
+        <span class="<%= filterButtonClass %>"></span>\
+      </div>\
+    ', null, DEFAULT_JST_DELIMS),
+
+    filterMenuTemplate : _.template('\
+      <div class="filterMenu dropdown-menu <%= listClass %>">\
+        <% if (filter.type === "string") { %>\
+          <input class="filter-string" id ="value" type="text" placeholder="Search <%= title %>" />\
```

Comment:
did you mean `<%= value %>` up above, because right now weren't you just setting id to the literal string `value`?  Trying to figure out exactly what you needed to persist. I agree a data-* attribute would be much better. I would also name the attribute something clear to indicate what it is storing.